helper-cli/src/main/kotlin/commands/GenerateScopeExcludesCommand.kt

```diff
@@ -89,138 +89,138 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
         "Bower" -> listOf(
             ScopeExclude(
                 pattern = "devDependencies",
-                reason = ScopeExcludeReason.BUILD_DEPENDENCY_OF,
-                comment = "Development dependencies."
+                reason = ScopeExcludeReason.DEV_DEPENDENCY_OF,
+                comment = "Scope with dependencies only used for development. Not included in released artifacts."
```

Comment:
I don't like the comments because what you add is redundant with the semantics of the scope exclude itself:
- The existance of a scope exclude means that that scope is not included in the release artifact.
- A scope exclude implies that it is about 'Scopes', so there is no need to add that prefix.

When reviewing ort.yml file all that has to be read and I `personally dislike` reading verbose autogenerated comments.